src/transfer/transfer.page.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useEffect } from "react";
+import {
+  Divider,
+  Grid,
+  CircularProgress,
+  Paper,
+  Box,
+  TableContainer,
+  Table,
+  TableBody,
+  TableHead,
+  TableRow,
+  Button,
+  TableFooter,
+  TablePagination,
+} from "@mui/material";
+import { Add } from "@mui/icons-material";
+import { users } from "../data/user";
+import { StyledTableCell, StyledTableRow } from "./transfer.styles";
+import { useNavigate } from "react-router-dom";
+
+export const Transfer = () => {
+  const [page, setPage] = React.useState(0);
+  const [rowsPerPage, setRowsPerPage] = React.useState(5);
+  const handleChangePage = (
+    event: React.MouseEvent<HTMLButtonElement> | null,
+    newPage: number
+  ) => {
+    setPage(newPage);
+  };
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
```

Comment:
you haven't done this in your previous PRs, please use the same convention for all PRs. If you stick to one, keep at it.

Reply:
Also, why are you using a bitwise or?

Reply:
For defining types I have always used | instead of || and I don't remember doing || for other type definitions. Also, I do this because the documentations that I do have read use only | and not ||.

Reply:
Sorry, I confused typescript syntax with something else.